packages/odyssey-react-mui/src/Badge.tsx

```diff
@@ -0,0 +1,52 @@
+/*!
+ * Copyright (c) 2023-present, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
+import { ReactNode, memo } from "react";
+import { Badge as MuiBadge, BadgeProps as MuiBadgeProps } from "@mui/material";
+import type { SeleniumProps } from "./SeleniumProps";
+
+export const badgeTypeValues = ["default", "primary", "error"] as const;
+export const badgeVariantValues = ["dot", "standard"] as const;
+
+export type BadgeProps = {
+  children?: ReactNode;
+  badgeContent?: number;
+  max?: MuiBadgeProps["max"];
```

Comment:
Let's find a more explicit name for `max`, it wasn't immediately clear to me that this was the max number before truncation.

Reply:
@jordankoschei-okta Can do. `max` is the prop that MUI uses by default so, just brought that over. Wasn't sure if we tried to carry prop names over from MUI or not. Any suggestions?

Reply:
@jordankoschei-okta Went with [badgeContentMax](https://github.com/okta/odyssey/pull/2005/commits/d1448d0bf81ca2be46e82dc2b66ecc015a36bcc9). Seem ok to you? Def open to suggestion still. Not in love with that name

Reply:
What about `maxNumberOfCharacters`?